packages/components/src/Form/Form.test.tsx

```diff
@@ -42,20 +42,20 @@ test("renders an error message when field is invalid", async () => {
   );
 });
 
-test("fires onStateChage when component renders", async () => {
+test("fires onStateChange when component renders", async () => {
   const stateChangeHandler = jest.fn();
   render(<MockForm onSubmit={jest.fn()} onStateChange={stateChangeHandler} />);
 
   await waitFor(() => {
     expect(stateChangeHandler).toHaveBeenCalled();
     expect(stateChangeHandler).toHaveBeenCalledWith({
       isDirty: false,
-      isValid: true,
+      isValid: false,
```

Comment:
`react-hook-form` updated how state initially fires - this change appears to have come from [this PR](https://github.com/react-hook-form/react-hook-form/pull/5415).

Now when the form renders, `isDirty` and `isValid` both render `false` as they should

Reply:
Pretty sure this will need to be updated too https://github.com/GetJobber/atlantis/blob/update-react-hook-form-round-two/packages/hooks/src/useFormState/useFormState.ts#L6

Reply:
Oo good call out!  Thank you!